### ISSUE TITLE
Feature Request - Game Launcher exit/close method

### ISSUE THREAD

**Issue by adamkrawczyk** [o3de/o3de#17122]
**Description**
When the game launcher is started there is no clear way to close it. The preferred way would be to press `Esc` key similar to the Editor.

Platform: 
Ubuntu 22

**Comment by lumberyard-employee-dm** [comment]
Perhaps for the Game Launcher this should be defaulted to Alt +F4. 
The Escape key is too easy to hit and exit the Game Launcher.

Also this features seems to fall in the purview of the Game developer themselves, as they should get the ultimate control over the application quit method (in-case they want to bring up a save dialog, etc...).

That being said, perhaps a pre-configured lua script or ScriptCanvas graph that binds a keyboard combination to the `quit` console command can be authored and added in the engine assets [folder](https://github.com/o3de/o3de/tree/development/Assets/Engine/Scripts/Utils)

**Comment by adamkrawczyk** [comment]
Yes, the `Alt + f4` works (but sometimes not everything is closed and the terminal hangs). The ideal solution would be to have a field in the `autoexec` file where will be possible to put the desired quit key/key combination.
